sdk/core/azure-core/src/http/log_policy.cpp

```diff
@@ -124,14 +61,16 @@ inline std::string GetResponseLogMessage(
       << "ms) : " << static_cast<int>(response.GetStatusCode()) << " "
       << response.GetReasonPhrase();
 
-  AppendHeaders(log, response.GetHeaders(), options.AllowedHttpHeaders);
+  AppendHeaders(log, inputSanitizer, response.GetHeaders());
   return log.str();
 }
 } // namespace
 
 Azure::Core::CaseInsensitiveSet const
     Azure::Core::Http::Policies::_detail::g_defaultAllowedHttpHeaders
-    = {"x-ms-request-id",
+    = {"traceparent",
```

Comment:
`"traceparent"` appears twice now. Remove it, and move `"tracestate"` next to an existing one.